helpers.js

```diff
@@ -0,0 +1,55 @@
+const log4js  = require('log4js');
+const LOG     = log4js.getLogger('helpers');
+const format  = require('format');
+const _       = require('lodash');
+
+// Using getMessages will automatically use already cached messages when available,
+//     and will fetch more messages only if needed. This way we don't have to rely
+//     on the promise function structure every time we want to retrieve a message
+exports.getMessages = async function getMessages(message) {
+
+  // Current messages in the Discord client's cache
+  let cachedMessages = message.channel.messages.array()[0].channel.messages;
+  
+  // If there are only a couple of items in the cache...
+  if (cachedMessages.array().length < 10) {
```

Comment:
do messages fall out of that cache?

Reply:
Not sure if/when messages fall out; this check in particular is arbitrary and just to backfill if it hasn't seen many messages yet